### PR TITLE
update imports so journeys-admin works locally

### PR DESCRIPTION
**Pull request by tataihono**
- remove theme provider on each journeysAdmin page
- update nextjs to v12.0.7
- enable swc on journeysAdmin
- fix handleAction error

### CODE REVIEW

File: apps/journeys-admin/next-env.d.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```
Comment (tataihono) [inline apps/journeys-admin/next-env.d.ts:2]: happened automatically when updating nextjs

File: apps/journeys-admin/jest.config.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   preset: '../../jest.preset.js',
   transform: {
     '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nrwl/react/plugins/jest',
-    '^.+\\.[tj]sx?$': 'babel-jest'
```
Comment (tataihono) [inline apps/journeys-admin/jest.config.js:6]: required as babel js file was removed to support swc for nextjs.

**Review by mikeallisonJS — APPROVED** [review]
Looks good

File: apps/journeys/src/components/blocks/Card/Card.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import {
   GetJourney_journey_blocks_ImageBlock as ImageBlock,
   GetJourney_journey_blocks_VideoBlock as VideoBlock
 } from '../../../../__generated__/GetJourney'
-import { CardCover } from '.'
+import { Cover as CardCover } from './Cover'
```
Comment (tataihono) [inline apps/journeys/src/components/blocks/Card/Card.tsx:13]: lint error with circular dependency.

**Pull request merged**